code/game/objects/structures/barsigns.dm

```diff
@@ -246,7 +246,7 @@
 /datum/barsign/emergencyrumparty
 	name = "The Emergency Rum Party"
 	icon = "emergencyrumparty"
-	desc = "Still serving drinks that were banned years ago."
+	desc = "Recently relicenced after a long closure."
```

Comment:
@duncathan that's not strictly true.
There's no rule on whether you use the wrong one (american) or the correct one (british)